### PR TITLE
Get count fsm

### PR DESCRIPTION
**Pull request by saurabh-egov**

### CODE REVIEW

**Review by harit-egov — APPROVED** [review]

File: packages/libraries/src/hooks/fsm/useApplicationStatus.js [hunk 1]
```diff
@@ -2,19 +2,24 @@ import React from "react";
...
+const useApplicationStatus = (select) => {
   const { t } = useTranslation();
   const tenantId = Digit.ULBService.getCurrentTenantId();
-  return useQuery("APPLICATION_STATUS", async () => {
+  const fetch = async () => {
     let WorkflowService = await Digit.WorkflowService.init(tenantId, "FSM");
+    return WorkflowService;
+  };
+  const defaultSelect = (WorkflowService) => {
+    console.log("defaultSelect executed");
```
Comment (saurabh-egov) [inline packages/libraries/src/hooks/fsm/useApplicationStatus.js:13]: remove

File: packages/libraries/src/hooks/fsm/useInbox.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-import useSearchAll from "./useSearchAll";
 
-const useInbox = (tenantId, filters) => {
-  const client = useQueryClient();
+const useInbox = (tenantId, filters, filterFsmFn, workFlowConfig = {}) => {
   let { uuid } = Digit.UserService.getUser().info;
-  const fetchApplications = ({ queryKey }) => {
-    const [_key, filters] = queryKey;
+
+  console.log("inside fetchInbox", filters);
```
Comment (saurabh-egov) [inline packages/libraries/src/hooks/fsm/useInbox.js:7]: remove

File: packages/libraries/src/services/molecules/FSM/DsoDetails.js [hunk 3]
```diff
@@ -4,8 +4,10 @@ const DsoDetails = async (tenantId, filters = {}) => {
   const dsoDetails = await FSMService.vendorSearch(tenantId, filters);
 
   //TODO get possible dates to book dso
+  console.log("dso details here", dsoDetails);
```
Comment (saurabh-egov) [inline packages/libraries/src/services/molecules/FSM/DsoDetails.js:7]: remove

File: packages/modules/fsm/src/pages/employee/ApplicationDetails/index.js [hunk 4]
```diff
@@ -68,6 +68,10 @@ const ApplicationDetails = (props) => {
     }
   }, [showToast]);
 
+  useEffect(() => {
+    console.log("applicationDetails here", applicationDetails);
```
Comment (saurabh-egov) [inline packages/modules/fsm/src/pages/employee/ApplicationDetails/index.js:72]: !! remove

**Pull request merged**